api/models/process.go

```diff
@@ -247,32 +249,32 @@ func ListProcesses(app string) ([]*Process, error) {
 		select {
 		case ps := <-psch:
 			pss = append(pss, &ps)
-		case err := <-errch:
-			return nil, err
 		}
 	}
 
 	pending, err := ListPendingProcesses(app)
-
 	if err != nil {
-		return nil, err
+		fmt.Printf("ns=kernel at=ListProcesses state=error message=\"unable to get pending processes: %s\"\n", err)
 	}
 
 	pss = append(pss, pending...)
 
 	oneoff, err := ListOneoffProcesses(app)
 
 	if err != nil {
-		return nil, err
+		fmt.Printf("ns=kernel at=ListProcesses state=error message=\"unable to get one-off processes: %s\"\n", err)
```

Comment:
This looks right. This should definitely be a best effort addition to the process list and not cause a total error.